.github/workflows/go.yml

```diff
@@ -33,3 +34,12 @@ jobs:
 
     - name: Test
       run: make test
+
+    strategy:
+      matrix:
+        go:
+          - 1.15
+          - 1.16
+          - 1.17
+        platform:
+          - ubuntu-latest
```

Comment:
🌷🌷 maybe a bit dangerous in case of a BC break from Ubuntu: everytime that we don't pin a version somewhere, we end up paying a high price

Reply:
@gyndav in that case I'm happy to see the break in the CI to spot it faster :)